packages/front-end/components/Avatar/InitialAvatar.tsx

```diff
@@ -0,0 +1,16 @@
+import Avatar, { Props as AvatarProps } from "@/components/Radix/Avatar";
+
+type Props = {
+  name: string;
+} & Omit<AvatarProps, "children">;
+
+export default function InitialAvatar({ name, ...otherProps }: Props) {
```

Comment:
not sure this should be a separate component. Imagine at some point we allow avatar customization. You'd need to conditionally choose the component type everywhere this is implemented. Maybe instead use a helper function `getUserAvatar()` that either returns the avatar icon or defaults to the user initials.

Reply:
I originally did this inside of our Radix Avatar component, but I think I'd need to change its props. Right now that component takes in children for its content and that would be fine if the avatar was for the current user, but in this case it's the experiment owner so we'd likely need to pass in name as a prop. If we create a helper function like `getUserAvatar()` then presumably we don't want it to take in `children` anyway, but it seems like we're also using the Avatar component for an icon in `EventSourceList` right now. Maybe we keep the HOC if we don't want to change our Radix Avatar wrapper and give it the helper function or we just don't use Avatars for icons and I can remove the `children` prop

Reply:
can you make a <UserAvatar> HOC that takes in necessary user info and then makes a decision about what to pass as `children` to <Avatar>?

Reply:
maybe props look like...
```
{
  name: string,
  icon?: ReactElement,
  color?: RadixColor,
}
```

Reply:
Kept the `Avatar` props and excluded `children` and added `name` and `icon`. Not sure if we have a `radius` and `variant` that we plan to use for all user avatars.

Reply:
maybe I'm just hung up on the name? I'd expect this to be the definitive user avatar component, not just the "initial" component used when we don't have enough information to render a proper avatar

Reply:
Oh sorry I renamed the component and not the file 🤦🏻‍♂️

Reply:
Fixed